apps/full-site-editing/bin/sync-newspack-blocks.sh

```diff
@@ -124,9 +124,8 @@ cp -R $CODE/src/blocks/carousel $TARGET/blocks/
 cp -R $CODE/src/shared $TARGET/
 cp -R $CODE/src/components $TARGET/
 
-# Replace text domain
-find $TARGET/blocks/ \( -name \*.js -or -name \*.php \) -exec sed "${sedi[@]}" "s/, 'newspack-blocks' )/, 'full-site-editing' )/g" "{}" \;
-sed "${sedi[@]}" "s/'newspack-blocks',/'full-site-editing',/g" $TARGET/class-newspack-blocks.php
+# Fix the text domain
+npx eslint . --fix
```

Comment:
I tried these things quickly but didn't get it working as I wanted:
```
npx eslint \
	--ext .js,.jsx,.ts,.tsx \
	--rule "'@wordpress/i18n-text-domain': ['error', {allowedTextDomain: 'full-site-editing'}]" \
	--fix \
	"$TARGET"
```

```
npx eslint \
	--ext .js,.jsx,.ts,.tsx \
	--no-eslintrc \
	--parser-options=sourceType:module \
	--plugin "@wordpress/eslint-plugin" \
	--rule "'@wordpress/i18n-text-domain': ['error', {allowedTextDomain: 'full-site-editing'}]" \
	--fix \
	"$TARGET"
```

Parser options still needs "ecmaVersion:7" and "ecmaFeatures:jsx:true" but didn't look up how to pass additional options nor if it makes sense instead of relying on eslint config.

Plugin/rule names might also be incorrect since it kept complaining it cannot find those.

I also hoped to apply only this one rule, but `--rule` only _appends_ to other rules in Eslint config, which was a bummer. We don't need to modify the code in any other way than textdomains.

Only functioning addition we could still add is `--ext .js,.jsx,.ts,.tsx "$TARGET"` since that'll improve performance.


(for context, see convo at https://github.com/Automattic/wp-calypso/pull/43084#issuecomment-641056564)

Reply:
When I tried something like this, I stumbled upon https://github.com/IanVS/eslint-nibble but didn't try to use it.

Reply:
Why not just do it on `$TARGET` and not worry about any of the other stuff? Would that pick up on the root eslint config?